tests/README.md

```diff
@@ -121,9 +121,34 @@ This allows run by case filter to simulate CI. Anybody can customize the case la
   * Export the token
     * `export RHCS_TOKEN=<rhcs token>` #If it had been run, then you can skip
   * Export the label filter to locate the preparation case
-    * `export LabelFilter="destroy"` #This filter only choose all critical and high priority cases with day1-post and day2
+    * `export LabelFilter="destroy"` #This filter only choose the destroy case for resources destroy
   * Run make command
     * `make e2e_test`
+### To run on local simulate CI without make file
```

Comment:
Are the flows with  `make e2e_test` command still relevant?

Also it fails on • [PANICKED] [50.411 seconds]
TF Test Create cluster test [It] CreateClusterByProfile [day1-prepare]
/Users/amalykhi/repos/terraform-provider-rhcs/tests/e2e/cluster_creation.go:13

  [PANICKED] Test Panicked
  In [It] at: /opt/homebrew/Cellar/go/1.20.4/libexec/src/runtime/iface.go:262 @ 10/18/23 15:36:56.799

  interface conversion: interface {} is nil, not []interface {}

Reply:
Good to know you fixed it. Resolve it.